src/Components/Publishing/Header/__tests__/__snapshots__/ClassicHeader.test.tsx.snap

```diff
@@ -382,7 +382,6 @@ exports[`Classic Header renders classic header with children properly 1`] = `
     >
       <div
         className="c3 c4"
-        color={undefined}
```

Comment:
Not sure why, but looks like with tsc we were capturing all the `undefined` props as well 🤷‍♂️